### PR TITLE
Added links to settings page 

### PR DESCRIPTION
**Pull request by Shadyholic**
https://github.com/Qsilver97/Qwallet/issues/153  

Converted buttons links to anchor links and added their link to open in a new tab in browser.

### CODE REVIEW

File: app/client/src/pages/Settings/InnerSettings.tsx
Comment (RealWorld777) [inline app/client/src/pages/Settings/InnerSettings.tsx:1]: website url must be qsilver.org not qubic.org

Comment (RealWorld777) [reply app/client/src/pages/Settings/InnerSettings.tsx:1]: https://qsivler.org

and its better to delete privacy policy, terms of service and add following things
image

**Review by serendipity-seeker — CHANGES_REQUESTED** [review]

**Comment by Shadyholic** [comment]
updated

**Review by serendipity-seeker — APPROVED** [review]
Seems good.

**Pull request merged**